web/packages/extension/src/content.ts

```diff
@@ -42,8 +22,9 @@ const ID = Math.floor(Math.random() * 100000000000);
  */
 function sendMessageToPage(data: unknown): Promise<unknown> {
     const message = {
-        to: `ruffle_page${ID}`,
+        to: "ruffle_page",
         index: pendingMessages.length,
+        id: ID,
```

Comment:
The ID is now passed as part of the message, as it can't be part of a script src since content scripts don't have a src.